testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/AcquireTokenFragment.java

```diff
@@ -203,6 +205,23 @@ public void onClick(View v) {
             }
         });
 
+        mKnownBrokerApps = view.findViewById(R.id.spinner_knownBrokerApps);
+        mSetCachedActiveBrokerButton = view.findViewById(R.id.btn_setCachedActiveBroker);
+
+        final List<BrokerData> debugBrokers = new ArrayList<>(BrokerData.getDebugBrokers());
```

Comment:
Seems like all the brokers, regardless of what's currently installed, are added to the spinner. Are there specific cases where you'd want to force set an active broker which is not installed onto the device?